src/include/add-on/add-on-workflow.rb

```diff
@@ -1171,49 +1171,29 @@ def ReturnCurrentlySelectedProductInfo
     end
 
     def AdjustInfoWidget
-      pi = ReturnCurrentlySelectedProductInfo()
-      if pi.nil? || pi == {}
+      product_info = ReturnCurrentlySelectedProductInfo()
+
+      if product_info.to_h.empty?
         UI.ChangeWidget(Id("product_details"), :Value, "")
+
         return
       end
 
-      vendor = pi["product"].vendor.empty? ? _("Unknown vendor") : pi["product"].vendor
-      version = pi["product"].version.empty? ? _("Unknown version") : pi["product"].version
-      rt_description = Builtins.sformat(
-        "<p>%1\n%2\n%3\n%4</p>",
-        Builtins.sformat(
-          _("<b>Vendor:</b> %1<br>"),
-          vendor
-        ),
-        Builtins.sformat(
-          _("<b>Version:</b> %1<br>"),
-          version
-        ),
-        Builtins.sformat(
-          _("<b>Repository URL:</b> %1<br>"),
-          if Ops.greater_than(
-            Builtins.size(Ops.get_list(pi, ["info", "URLs"], [])),
-            0
-          )
-            Builtins.mergestring(Ops.get_list(pi, ["info", "URLs"], []), ",")
-          else
-            _("Unknown repository URL")
-          end
-        ),
-        if Ops.greater_than(
-          Builtins.size(Ops.get_list(pi, ["info", "aliases"], [])),
-          0
-        )
-          Builtins.sformat(
-            _("<b>Repository Alias:</b> %1<br>"),
-            Builtins.mergestring(Ops.get_list(pi, ["info", "aliases"], []), ",")
-          )
-        else
-          ""
-        end
-      )
+      product = product_info["product"]
+      info    = product_info["info"] || {}
+
+      vendor  = product.vendor.empty?  ? _("Unknown vendor")  : product.vendor
+      version = product.version.empty? ? _("unknown version") : product.version
+      urls    = info.fetch("URLs", []).join(",")
+      aliases = info.fetch("aliases", []).join(",")
+
+      details = []
+      details << format(_("<b>Vendor:</b> %s<br>"), vendor)
```

Comment:
I know this change a bit the translation since former `%1` is now `%s` but... I hope it does not hurt. If so, I can revert the last commit completely.

Reply:
Unfortunately in my experience the translators quite often overlook such small differences and they just reuse the old translations. :worried: 

But we have the [check_po_files.rb](https://github.com/yast/yast-translations/blob/master/tools/check/check_po_files.rb) for that reason. I run it from time to time and do fixes like [this](https://github.com/yast/yast-translations/commit/2a0d8433903a9d360e8e60e225b2564a528b2f0f).

So go ahead, I'll run the script after few days to fix the problems. :smiley: